src/styles/utils/variables.scss

```diff
@@ -0,0 +1,13 @@
+$bodyFont: Roboto, sans-serif;
+$defFont-W: 500;
+$defFont-S: 12px;
+$logoSize: 40px;
+$textColorMain: #060b35;
+$textColorSecondary: #616070;
+$navLinkColor: #060b35;
+$blueColor: #00acdc;
+$blackColor: #000;
+$whiteColor: #fff;
+$borderColor: #f3f3f3;
+$columnCount: 1;
+$rowCount: 2;
```

Comment:
I don't use the two last variables. I forgot to erase them from the code 😅

Reply:
so you can always make new commit and fix it there)